tools/common/StaticRegistrar.cs

```diff
@@ -3704,8 +3712,10 @@ public void Generate (IEnumerable<AssemblyDefinition> assemblies, string header_
 		{
 			this.input_assemblies = assemblies;
 
-			foreach (var assembly in assemblies)
+			foreach (var assembly in assemblies) {
+				Driver.Log (3, "Generating static registrar for {0}", assembly.Name);
```

Comment:
This is an unrelated change (wrt commit message). However you need to update it since it does not match the diff anymore so please also state the addition of the logging.